coriolisclient/cli/utils.py

```diff
@@ -114,3 +115,14 @@ def format_json_for_object_property(obj, prop_name):
         prop = prop.to_dict()
 
     return json.dumps(prop, indent=2)
+
+
+def validate_uuid(uuid_string):
+    try:
+        val = UUID(uuid_string, version=4)
```

Comment:
Please make `version` a kwarg whose default value is `4` in instead of hardcoding it here.